modules/core/src/webgl/features/webgl-features-table.js

```diff
@@ -1,35 +1,77 @@
+import {isWebGL2} from '../utils';
+
+// TODO - this should be the default export, test cases need updating
+export const FEATURES = {
+  WEBGL2: 'WEBGL2',
+
+  // API SUPPORT
+  VERTEX_ARRAY_OBJECT: 'VERTEX_ARRAY_OBJECT',
+  TIMER_QUERY: 'TIMER_QUERY',
+  INSTANCED_RENDERING: 'INSTANCED_RENDERING',
+  MULTIPLE_RENDER_TARGETS: 'MULTIPLE_RENDER_TARGETS',
+
+  // FEATURES
+  ELEMENT_INDEX_UINT32: 'ELEMENT_INDEX_UINT32',
+  BLEND_EQUATION_MINMAX: 'BLEND_EQUATION_MINMAX',
+
+  // TEXTURES: '// TEXTURES', RENDERBUFFERS
+  COLOR_ENCODING_SRGB: 'COLOR_ENCODING_SRGB',
+
+  // TEXTURES
+  TEXTURE_DEPTH: 'TEXTURE_DEPTH',
+  TEXTURE_FLOAT: 'TEXTURE_FLOAT',
+  TEXTURE_HALF_FLOAT: 'TEXTURE_HALF_FLOAT',
+
+  TEXTURE_FILTER_LINEAR_FLOAT: 'TEXTURE_FILTER_LINEAR_FLOAT',
+  TEXTURE_FILTER_LINEAR_HALF_FLOAT: 'TEXTURE_FILTER_LINEAR_HALF_FLOAT',
+  TEXTURE_FILTER_ANISOTROPIC: 'TEXTURE_FILTER_ANISOTROPIC',
+
+  // FRAMEBUFFERS: '// FRAMEBUFFERS', TEXTURES AND RENDERBUFFERS
+  COLOR_ATTACHMENT_RGBA32F: 'COLOR_ATTACHMENT_RGBA32F',
+  COLOR_ATTACHMENT_FLOAT: 'COLOR_ATTACHMENT_FLOAT',
+  COLOR_ATTACHMENT_HALF_FLOAT: 'COLOR_ATTACHMENT_HALF_FLOAT',
+
+  // GLSL extensions
+  GLSL_FRAG_DATA: 'GLSL_FRAG_DATA',
+  GLSL_FRAG_DEPTH: 'GLSL_FRAG_DEPTH',
+  GLSL_DERIVATIVES: 'GLSL_DERIVATIVES',
+  GLSL_TEXTURE_LOD: 'GLSL_TEXTURE_LOD'
+};
+
 // Defines luma.gl "feature" names and semantics
 export default {
+  [FEATURES.WEBGL2]: [gl => isWebGL2(gl)],
+
   // API SUPPORT
-  VERTEX_ARRAY_OBJECT: ['OES_vertex_array_object', true],
-  TIMER_QUERY: ['EXT_disjoint_timer_query', 'EXT_disjoint_timer_query_webgl2'],
-  INSTANCED_RENDERING: ['ANGLE_instanced_arrays', true],
-  MULTIPLE_RENDER_TARGETS: ['WEBGL_draw_buffers', true],
+  [FEATURES.VERTEX_ARRAY_OBJECT]: ['OES_vertex_array_object', true],
+  [FEATURES.TIMER_QUERY]: ['EXT_disjoint_timer_query', 'EXT_disjoint_timer_query_webgl2'],
```

Comment:
Make the WebGL 2 version both. Nice use case for your new array functionality.

Reply:
> Make the WebGL 2 version both. Nice use case for your new array functionality.

Yes, let's do it separately. For Firefox, so needs separate testing, especially given how this extension has been flipped on and off by browsers.

Reply:
You just want to make sure the array code path is actually being used?

Reply:
Yes. Not a huge deal here, but building out logic that isn't in used anywhere triggers an allergic reaction in me. Can become fertile ground for bit rot and confusion about intention.